content/content-strategy/information-architecture/index.yml

```diff
@@ -1,4 +1,4 @@
-pagetitle: Information architecture
+pagetitle: Information architecture (IA)
 weight: 10
```

Comment:
May be hidden but anticipate where it's gonna sit in the IA and please adjust this value accordingly. Recommend: compare weight values across the module *and* parent, Content strategy so that IA will appear correctly in the menu when the content is shown.

Reply:
To be revisited later.